docs/outlook/autolaunch.md

```diff
@@ -490,7 +490,14 @@ Some Office.js APIs that change or alter the UI aren't allowed from event-based
   - `displayDialogAsync`
   - `messageParent`
 
-### Requesting external data
+### Preview features in event handlers (Outlook on Windows)
+
+To test [Outlook features that are in preview](/javascript/api/requirement-sets/outlook/preview-requirement-set/outlook-requirement-set-preview) in event handlers on Outlook on Windows, you must configure your computer's registry as follows.
```

Comment:
In the past, we've been asked to explain why this is the case. Consider incorporating info from [this PR](https://github.com/OfficeDev/office-js-docs-pr/pull/2690/files).